includes/functions/metadata.php

```diff
@@ -485,6 +507,25 @@ function resource_data_init() {
 		]
 	);
 
+	$cmb->add_field(
+		[
+			'name'        => __( 'Publication Name', 'learning-commons-framework' ),
+			'description' => __( 'The publication in which the resource appears.', 'learning-commons-framework' ),
+			'id'          => $prefix . 'publication_name',
+			'type'        => 'text',
+		]
+	);
+
+	$cmb->add_field(
+		[
+			'name'        => __( 'Publication Link', 'learning-commons-framework' ),
+			'description' => __( 'A link to the publication in which the resource appears.', 'learning-commons-framework' ),
+			'id'          => $prefix . 'publication_link',
+			'type'        => 'text_url',
+			'protocols'   => [ 'http', 'https' ],
```

Comment:
Not specific to this PR, but validation of URLs in general - an URL can be ill formed and successfully submitted.

Example: http://foo
Example: http://foo-

Reply:
@jhung Can you open a separate issue for this?